src/App.tsx

```diff
@@ -1,24 +1,391 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import * as postService from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 10876;
+
+enum Filter {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<Filter>(Filter.ALL);
+  const [isShowError, setIsShowError] = useState<boolean>(false);
+  const [error, setError] = useState<string>('');
```

Comment:
looks like redundant isShowError state
Enought error state and you can  reset this state when error shouldn't render